dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.json

```diff
@@ -0,0 +1,1060 @@
+{
+ "title": "converted from jotform",
+ "pages": [
+  {
+   "name": "page 1",
+   "elements": [
+    {
+     "type": "text",
+     "name": "userId",
+     "visible": false,
+     "title": "userId"
+    },
+    {
+     "type": "text",
+     "name": "course",
+     "visible": false,
+     "title": "course"
+    },
+    {
+     "type": "text",
+     "name": "pd",
+     "visible": false,
+     "title": "pd"
+    },
+    {
+     "type": "html",
+     "name": "clickTo57",
+     "html": "<p><span style=\"font-size: 18pt;\"><strong>[DRAFT 05.29.20]Welcome to the 2019-20 {course} post-course survey</strong></span></p>\n<p>Code.org relies on surveys as a crucial tool through we remain connected to teachers, and to give you the opportunity to provide feedback about our curriculum and your teaching experience.</p>\n<p><strong>Privacy and Confidentiality</strong></p>\n<ul>\n<li>Your individual responses here are completely <strong>confidential.</strong></li>\n<li>Your identity will <strong>never</strong> be revealed publicly or used to tie any responses back to you personally.</li>\n<li>We may share <strong>anonymous,</strong> aggregate data (representing all responses) publicly.</li>\n<li>Your <strong>anonymized</strong> responses will be used by Code.org, our Regional Partners, and our facilitators for program improvement.</li>\n</ul>\n<p><strong>This survey should take less than 15 minutes to complete. </strong></p>\n<p><strong>At the end you will be offered at $10 Amazon gift code for your time (supplies limited)</strong></p>\n<p><strong>Your honesty and candor are appreciated.<br /></strong></p>"
+    },
+    {
+     "type": "html",
+     "name": "Preamble",
+     "html": "<p style=\"text-align: left;\"><strong><span style=\"font-size: 18pt;\">Teaching Context </span></strong></p>\n<p>This survey is about teaching with Code.org materials and resources over the entire 2019-20 academic year.</p>\n<p>We address teaching the course under &quot;normal&quot; circumstances, as well as remotely.</p>\n<p> </p>"
+    },
+    {
+     "type": "checkbox",
+     "name": "grade_levels_taught",
+     "title": "At what grade levels did you teach {course}? [select all that apply]",
+     "choices": [
+      "K",
```

Comment:
choices should be in object format (same for questions below)

Reply:
gah. thought I fixed that too.  Okay. will investigate.